browser/brave_browser_process_impl.cc

```diff
@@ -246,6 +258,14 @@ BraveBrowserProcessImpl::tor_client_updater() {
       brave_component_updater_delegate());
   return tor_client_updater_.get();
 }
+
+void BraveBrowserProcessImpl::OnTorEnabledChanged() {
+  // Update all browsers' tor command status.
+  for (Browser* browser : *BrowserList::GetInstance()) {
```

Comment:
When tor option is changed, all browsers' tor commands are updated.

Reply:
does the utility process automatically shut down when all tor windows are closed?

Reply:
In nightly, tor process is gone. but not in stable version.